pkg/iac/scanners/terraform/ignore_test.go

```diff
@@ -433,7 +433,7 @@ resource "bad" "my-rule" {
 		{
 			name: "ignore by dynamic block value",
 			inputOptions: `
-// trivy:ignore:*[secure_settings.0.enabled=false]
+// trivy:ignore:*[secure_settings.enabled=false]
```

Comment:
let's add these two as separate test cases so they're both documented in behavior.

Reply:
Done https://github.com/aquasecurity/trivy/pull/7205/commits/d71e6ac6bbc462df34e31e36debe6a903f534616